<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Finanzas</class>
 <widget class="QMainWindow" name="Finanzas">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>865</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Finanzas</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frame">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QFrame" name="frame_2">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QTabWidget" name="tabWidget">
            <property name="currentIndex">
             <number>0</number>
            </property>
            <widget class="QWidget" name="tab_3">
             <attribute name="title">
              <string>Gasto Semanal</string>
             </attribute>
             <widget class="QGroupBox" name="groupBox">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>330</y>
                <width>811</width>
                <height>211</height>
               </rect>
              </property>
              <property name="title">
               <string>Gráfico de Gastos Semanales</string>
              </property>
             </widget>
             <widget class="QTableWidget" name="tablaInput">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>70</y>
                <width>811</width>
                <height>211</height>
               </rect>
              </property>
              <property name="sizeAdjustPolicy">
               <enum>QAbstractScrollArea::AdjustToContents</enum>
              </property>
              <property name="autoScroll">
               <bool>false</bool>
              </property>
              <property name="editTriggers">
               <set>QAbstractItemView::AnyKeyPressed|QAbstractItemView::CurrentChanged|QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed</set>
              </property>
              <property name="dragDropMode">
               <enum>QAbstractItemView::DragOnly</enum>
              </property>
              <property name="alternatingRowColors">
               <bool>true</bool>
              </property>
              <property name="selectionMode">
               <enum>QAbstractItemView::SingleSelection</enum>
              </property>
              <property name="showGrid">
               <bool>true</bool>
              </property>
              <property name="gridStyle">
               <enum>Qt::DashLine</enum>
              </property>
              <property name="sortingEnabled">
               <bool>true</bool>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
              <property name="cornerButtonEnabled">
               <bool>true</bool>
              </property>
              <attribute name="horizontalHeaderCascadingSectionResizes">
               <bool>false</bool>
              </attribute>
              <attribute name="verticalHeaderShowSortIndicator" stdset="0">
               <bool>false</bool>
              </attribute>
              <row>
               <property name="text">
                <string>Transporte</string>
               </property>
              </row>
              <row>
               <property name="text">
                <string>Comida</string>
               </property>
              </row>
              <row>
               <property name="text">
                <string>Educación</string>
               </property>
              </row>
              <row>
               <property name="text">
                <string>Entretenimiento</string>
               </property>
              </row>
              <row>
               <property name="text">
                <string>Ahorro</string>
               </property>
              </row>
              <row>
               <property name="text">
                <string>Otro</string>
               </property>
              </row>
              <column>
               <property name="text">
                <string>Lunes</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Martes</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Miercoles</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Jueves</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Viernes</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Sábado</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Domingo</string>
               </property>
              </column>
              <item row="0" column="0">
               <property name="text">
                <string extracomment="0000"/>
               </property>
              </item>
              <item row="0" column="1">
               <property name="text">
                <string/>
               </property>
              </item>
              <item row="0" column="2">
               <property name="text">
                <string/>
               </property>
              </item>
              <item row="0" column="3">
               <property name="text">
                <string/>
               </property>
              </item>
              <item row="1" column="0">
               <property name="text">
                <string/>
               </property>
              </item>
              <item row="1" column="1">
               <property name="text">
                <string/>
               </property>
              </item>
              <item row="1" column="2">
               <property name="text">
                <string/>
               </property>
              </item>
              <item row="1" column="3">
               <property name="text">
                <string/>
               </property>
              </item>
              <item row="1" column="4">
               <property name="text">
                <string/>
               </property>
              </item>
              <item row="1" column="5">
               <property name="text">
                <string/>
               </property>
              </item>
              <item row="2" column="0">
               <property name="text">
                <string/>
               </property>
              </item>
              <item row="2" column="1">
               <property name="text">
                <string/>
               </property>
              </item>
              <item row="2" column="2">
               <property name="text">
                <string/>
               </property>
              </item>
              <item row="2" column="3">
               <property name="text">
                <string/>
               </property>
              </item>
              <item row="2" column="4">
               <property name="text">
                <string/>
               </property>
              </item>
              <item row="2" column="5">
               <property name="text">
                <string/>
               </property>
              </item>
              <item row="3" column="0">
               <property name="text">
                <string/>
               </property>
              </item>
              <item row="3" column="1">
               <property name="text">
                <string/>
               </property>
              </item>
              <item row="3" column="2">
               <property name="text">
                <string/>
               </property>
              </item>
              <item row="3" column="3">
               <property name="text">
                <string/>
               </property>
              </item>
              <item row="3" column="4">
               <property name="text">
                <string/>
               </property>
              </item>
              <item row="3" column="5">
               <property name="text">
                <string/>
               </property>
              </item>
              <item row="4" column="0">
               <property name="text">
                <string/>
               </property>
              </item>
              <item row="4" column="1">
               <property name="text">
                <string/>
               </property>
              </item>
              <item row="4" column="2">
               <property name="text">
                <string/>
               </property>
              </item>
              <item row="4" column="3">
               <property name="text">
                <string/>
               </property>
              </item>
              <item row="4" column="4">
               <property name="text">
                <string/>
               </property>
              </item>
              <item row="4" column="5">
               <property name="text">
                <string/>
               </property>
              </item>
              <item row="5" column="0">
               <property name="text">
                <string/>
               </property>
              </item>
              <item row="5" column="1">
               <property name="text">
                <string/>
               </property>
              </item>
              <item row="5" column="2">
               <property name="text">
                <string/>
               </property>
              </item>
              <item row="5" column="3">
               <property name="text">
                <string/>
               </property>
              </item>
              <item row="5" column="4">
               <property name="text">
                <string/>
               </property>
              </item>
              <item row="5" column="5">
               <property name="text">
                <string/>
               </property>
              </item>
             </widget>
             <widget class="QLabel" name="label">
              <property name="geometry">
               <rect>
                <x>20</x>
                <y>10</y>
                <width>61</width>
                <height>31</height>
               </rect>
              </property>
              <property name="text">
               <string>Ingresos:</string>
              </property>
             </widget>
             <widget class="QDoubleSpinBox" name="doubleSpinBox">
              <property name="geometry">
               <rect>
                <x>80</x>
                <y>10</y>
                <width>141</width>
                <height>31</height>
               </rect>
              </property>
              <property name="decimals">
               <number>2</number>
              </property>
              <property name="maximum">
               <double>10000000.990000000223517</double>
              </property>
              <property name="stepType">
               <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
              </property>
             </widget>
             <widget class="QLabel" name="label_2">
              <property name="geometry">
               <rect>
                <x>230</x>
                <y>10</y>
                <width>21</width>
                <height>31</height>
               </rect>
              </property>
              <property name="text">
               <string>$</string>
              </property>
             </widget>
             <widget class="QPushButton" name="btnAplicar">
              <property name="geometry">
               <rect>
                <x>380</x>
                <y>300</y>
                <width>75</width>
                <height>24</height>
               </rect>
              </property>
              <property name="styleSheet">
               <string notr="true">#btnAplicar{
	color: rgb(255, 255, 255);
	background-color: rgb(85, 170, 255);
	border-radius: 3px;	
}

#btnAplicar:hover{
	background-color: rgb(71, 151, 255);
}</string>
              </property>
              <property name="text">
               <string>Aplicar</string>
              </property>
             </widget>
             <widget class="QGroupBox" name="groupBox_2">
              <property name="geometry">
               <rect>
                <x>600</x>
                <y>0</y>
                <width>211</width>
                <height>51</height>
               </rect>
              </property>
              <property name="title">
               <string>Egreso</string>
              </property>
              <widget class="QLabel" name="label_3">
               <property name="geometry">
                <rect>
                 <x>10</x>
                 <y>20</y>
                 <width>49</width>
                 <height>16</height>
                </rect>
               </property>
               <property name="text">
                <string>0$</string>
               </property>
              </widget>
             </widget>
             <widget class="QGroupBox" name="groupBox_3">
              <property name="geometry">
               <rect>
                <x>370</x>
                <y>0</y>
                <width>211</width>
                <height>51</height>
               </rect>
              </property>
              <property name="title">
               <string>Ahorros</string>
              </property>
              <widget class="QLabel" name="label_4">
               <property name="geometry">
                <rect>
                 <x>10</x>
                 <y>20</y>
                 <width>49</width>
                 <height>16</height>
                </rect>
               </property>
               <property name="text">
                <string>0$</string>
               </property>
              </widget>
             </widget>
             <widget class="QLabel" name="aviso_txt">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>50</y>
                <width>351</width>
                <height>16</height>
               </rect>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </widget>
            <widget class="QWidget" name="tab_4">
             <attribute name="title">
              <string>Tab 2</string>
             </attribute>
            </widget>
            <widget class="QWidget" name="tab_5">
             <attribute name="title">
              <string>Page</string>
             </attribute>
            </widget>
            <widget class="QWidget" name="tab_6">
             <attribute name="title">
              <string>Page</string>
             </attribute>
            </widget>
            <widget class="QWidget" name="tab_7">
             <attribute name="title">
              <string>Page</string>
             </attribute>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
